index.js

```diff
@@ -179,7 +136,7 @@ async function loadImages(total) {
   for (let i = 0; i < total; i++) {
     const num = numeral(i).format("0000");
     const img = await Jimp.read(
-      `AutoEncoder_TrainingData/data/square${num}.png`
+      `data/square${num}.png`
```

Comment:
Is there a reason you changed this? I like being able to pull the training data directly from the Processing sketch.

Reply:
no specific reason, i didn't have processing installed, so i created a Python script to generate the images, i only changed to the output of that